proxy/envoy/testdata/envoy-v0.json.golden

```diff
@@ -1,7 +1,53 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:1081",
+      "address": "tcp://0.0.0.0:80",
```

Comment:
Comments aren't allowed in JSON files so they would need to be stripped before parsing
(see https://plus.google.com/+DouglasCrockfordEsq/posts/RK8qyGVaGSr). Furthermore, the golden files are currently auto-generated so we'd need to annotate them by hand. Alternatively, we could start encoding test data directly in *_test.go files as JSON string or the unmarshaled golang struct equivalent, either of which may make it easier to document via comments, though it would be more tedious to update/maintain the golden data itself.